### ISSUE TITLE
VZO Indicator

### ISSUE THREAD

**Issue by ghost** [arkochhar/Technical-Indicators#15]
Have you ever thought to develop VZO indicator? @arkochhar

**Comment by arkochhar** [comment]
No, you can share the algorithm, and time permitting, I will look into it. Alternatively, you can develop it, and we can consider merging the implementation to this library.
